api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -186,6 +187,21 @@ private DbUser initializeUserIfNeeded() {
     return getProfileResponse(dbUser);
   }
 
+  @Override
+  public ResponseEntity<Boolean> getUserTerraTermsOfServiceStatus() {
```

Comment:
OK, i did not check that because i assumed the account creation was successful hence the AoU TOS has been accepted.
 I'll add the check and some tests